packages/stacking/src/index.ts

```diff
@@ -78,6 +78,51 @@ export interface BalanceInfo {
   nonce: number;
 }
 
+export interface RewardInfo {
+  canonical: boolean;
+  burn_block_hash: string;
+  burn_block_height: number;
+  burn_amount: number | string;
+  reward_recipient: string;
+  reward_amount: number | string;
+  reward_index: number;
+}
+
+export interface RewardHolderInfo {
+  canonical: boolean;
+  burn_block_hash: string;
+  burn_block_height: number;
+  address: string;
+  slot_index: number;
+}
+
+export interface RewardsResponse {
+  limit: number;
+  offset: number;
+  results: RewardInfo[];
+}
+
+export interface RewardHoldersResponse {
+  limit: number;
+  offset: number;
+  total: number;
+  results: RewardHolderInfo[];
+}
+
+export interface RewardsError {
+  error: string;
+}
+
+export interface RewardOptions {
+  limit: number;
+  offset: number;
+}
+
+export interface TotalRewardsResponse {
+  reward_recipient: string,
+  reward_amount: string | number;
+}
+
```

Comment:
Could these types be pulled in from the `@stacks/stacks-blockchain-api-types` package?

Reply:
Done